CHANGELOG.md

```diff
@@ -101,56 +118,80 @@ command either on the command line or in a [configuration
 file](https://certbot.eff.org/docs/using.html#configuration-file) to run
 an additional command before/after any certificate is renewed. Hooks
 will only be run if a certificate is renewed.
+* Support Busybox in certbot-auto.
+
+### Changed
+
 * Recategorized `-h/--help` output to improve documentation and
 discoverability.
-* Busybox support in certbot-auto.
+
+### Removed
+
+* Removed the ncurses interface. This change solves problems people
+were having on many systems, reduces the number of Certbot
+dependencies, and simplifies our code. Certbot's only interface now is
+the text interface which was available by providing `-t/--text` to
+earlier versions of Certbot.
+
+### Fixed
+
 * Many small bug fixes.
 
 More details about these changes can be found on our GitHub repo:
 https://github.com/certbot/certbot/issues?q=is%3Aissue+milestone%3A0.10.0is%3Aclosed
 
-# 0.9.3
-## 10/13/2016
+## 0.9.3 - 2016-10-13
+
+### Added
 
-* Adopt more conservative behavior about reporting a needed port as
-unavailable when using the standalone plugin.
 * The Apache plugin uses information about your OS to help determine the
 layout of your Apache configuration directory. We added a patch to
 ensure this code behaves the same way when testing on different systems
 as the tests were failing in some cases.
 
+### Changed
+
+* Adopt more conservative behavior about reporting a needed port as
```

Comment:
@SwartzCr good question, I dunno... the tenses in general are wildly inconsistent.